nabweb/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -215,69 +214,98 @@ msgstr ""
 "rel='noopener noreferrer'>GitHub repository</a>"
 msgstr ""
 "Le <a target='_blank' href='https://github.com/nabaztag2018/pynab/#readme' "
 "rel='noopener noreferrer'>référentiel GitHub</a>"
 
-#: templates/nabweb/help.html:19
+#: templates/nabweb/help.html:20
 msgid "Report bugs or contribute to the project with other developers."
 msgstr ""
 "Signalez des bogues ou contribuez au projet avec d'autres développeurs."
 
-#: templates/nabweb/help.html:20
+#: templates/nabweb/help.html:21
 msgid ""
 "<a target='_blank' href='https://twitter.com/nabaztagtagtag' rel='noopener "
 "noreferrer'>Nabaztag on Twitter</a>"
 msgstr ""
 "<a target='_blank' href='https://twitter.com/nabaztagtagtag' rel='noopener "
 "noreferrer'>Nabaztag sur Twitter</a>"
 
-#: templates/nabweb/help.html:21
+#: templates/nabweb/help.html:22
 msgid ""
 "The <a target='_blank' href='https://www.instructables.com/member/"
 "tagtagtag/' rel='noopener noreferrer'>TagTagTag assembly instructions</a>"
 msgstr ""
 "Les <a target='_blank' href='https://www.instructables.com/member/"
 "tagtagtag/' rel='noopener noreferrer'>instructions de montage TagTagTag</a>"
 
-#: templates/nabweb/help.html:21
+#: templates/nabweb/help.html:22
 msgid ""
 "The open heart operation to transplant the card into your rabbit. At this "
 "stage you probably do not need this anymore..."
 msgstr ""
 "L'opération à cœur ouvert pour transplanter la carte dans votre lapin. A ce "
 "stade, vous n'en avez probablement plus besoin ..."
 
-#: templates/nabweb/help.html:22
+#: templates/nabweb/help.html:23
+#, fuzzy
+#| msgid ""
+#| "<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-"
+#| "nabaztag/' rel='noopener noreferrer'>The eternal return of the Nabaztag</"
+#| "a>"
 msgid ""
-"The <a target='_blank' href='https://www.tagtagtag.fr/index_eng.html' "
-"rel='noopener noreferrer'>Eternal Sunshine of the Nabaztag</a>"
+"<a target='_blank' href='https://fr.ulule.com/le-retour-du-retour-du-"
+"nabaztag/' rel='noopener noreferrer'>Nabaztag is BackBack</a>"
 msgstr ""
-"L'<a target='_blank' href='https://www.tagtagtag.fr/index.html' "
-"rel='noopener noreferrer'>éternel retour du Nabaztag</a>"
+"<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' "
+"rel='noopener noreferrer'>L'éternel retour du Nabaztag</a>"
 
-#: templates/nabweb/help.html:22
-msgid "The TagtagTag project..."
-msgstr "Le projet TagTagTag ..."
+#: templates/nabweb/help.html:23
+msgid "Some project history..."
+msgstr "Un peu d'histoire ..."
+
+#: templates/nabweb/help.html:24
+msgid ""
+"<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' "
+"rel='noopener noreferrer'>The eternal return of the Nabaztag</a>"
+msgstr ""
+"<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' "
+"rel='noopener noreferrer'>L'éternel retour du Nabaztag</a>"
+
+#: templates/nabweb/help.html:24
+msgid "Collection 2021-2022"
+msgstr "Collection 2021-2022"
```

Comment:
I do **not** agree with this change, which basically reverts the changes that were made in PR https://github.com/nabaztag2018/pynab/pull/298 [remove direct references to Ulule campaigns (which come and go...) and replace them by a reference to the tagtagtag.fr project page (which itself references the Ulule campaigns)].